ni_measurementlink_service/_featuretoggles.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import annotations
+
+import functools
+import sys
+from enum import Enum
+from typing import TYPE_CHECKING, Any, Callable, TypeVar
+
+from decouple import config
+
+if sys.version_info >= (3, 10):
+    from typing import ParamSpec
+else:
+    from typing_extensions import ParamSpec
+
+if TYPE_CHECKING:
+    _F = TypeVar("_F", bound=Callable[..., Any])
+    _P = ParamSpec("_P")
+    _T = TypeVar("_T")
+
+
+class CodeReadiness(Enum):
+    """Indicates whether code is ready to be supported."""
+
+    RELEASE = 0
+    NEXT_RELEASE = 1
+    INCOMPLETE = 2
+    PROTOTYPE = 3
```

Comment:
This uses the same enum ordering as Core. grpc-device [reverses this ordering](https://github.com/ni/grpc-device/blob/main/source/server/feature_toggles.h).